src/services/Libs.ts

```diff
@@ -104,6 +104,33 @@ export const convertVersionToNumber = (version?: string): number => {
   return parseInt(version.replace(/[.]/g, ''), 10);
 };
 
+/**
+ * Abstract Event Listener call to add/remove with checks.
+ * @param event The event listener
+ * @param listener The callback function to add/check/remove.
+ * @param action The EventListenerAction (add/remove).
+ */
+export const eventListenerActions = (
+  event: EvListener<any>,
+  listener: (...args: any[]) => void,
+  action: EventListenerAction,
+): void => {
+  if (!event) return;
```

Comment:
You can probably combine these two `if (!event && !event.hasListener) return;`

Reply:
Combined via 4153672